packages/elements/src/autosuggest/index.ts

```diff
@@ -1404,7 +1378,63 @@ export class Autosuggest extends Overlay {
 
     event.stopPropagation();
     event.preventDefault();
-  };
+  }
+
+  /**
+   * @returns template of loader if currently query loading
+   */
+  protected get loaderTemplate (): TemplateResult | null {
+    if (!this.loading) {
+      return null;
+    }
+    return html`
+      <div part="loader">
+        <div part="backdrop"></div>
+        <ef-loade></ef-loader>
```

Comment:
```suggestion
        <ef-loader></ef-loader>
```

Reply:
Is there anything we need to do to make this loader accessible ?

Reply:
Maybe something like [this](https://v6.refinitiv-ui.pages.dev/elements/loader#accessibility) would do.